src/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonConverterEnum.cs

```diff
@@ -17,6 +18,10 @@ public override bool CanConvert(Type type)
             return type.IsEnum;
         }
 
+        [PreserveDependency(
```

Comment:
Did you try it without the attribute? The linker has detection for custom TypeConverters so this should work out of box

Reply:
I can double check, but I'm pretty sure things blew up horribly without this one.

Reply:
Taking this back. I didn't look at the implementation and thought this is about the actual converter type